packages/ra-ui-materialui/src/list/filter/FilterLiveSearch.tsx

```diff
@@ -23,10 +23,16 @@ import { TextInput } from '../../input';
  * );
  */
 export const FilterLiveSearch = memo((props: FilterLiveSearchProps) => {
-    const { source = 'q', variant, ...rest } = props;
     const { filterValues, setFilters } = useListFilterContext();
     const translate = useTranslate();
 
+    const {
+        source = 'q',
+        variant,
+        label = translate('ra.action.search'),
```

Comment:
You don't even need to translate here, just pass the translation key directly to the `label` prop. `<TextInput>` will handle the translation

Reply:
Nevermind, accepting react node for this label would complicate the code as we only use it as the `TextInput` label when the `variant` is `outlined`. Otherwise, we pass it as the `placeholder` prop